rfcs/0064-documentation-format.md

```diff
@@ -0,0 +1,173 @@
+---
+feature: documentation-format
+start-date: 2019-12-31
+author: Silvan Mosberger (infinisil)
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+The Nix community wants to move away from using Docbook as our documentation format and replace it with something else. However it is unclear what it should be replaced with. This RFC gives a concrete process for determining the new documentation format. It does NOT say what format should be used.
```

Comment:
I think this topic would be much better approached as “The Nix community wants to evaluate alternatives from Docbook”. Otherwise we could get into a situation where we’ve decided to move away, but no new format is any better than Docbook is, and then we’re stuck.

Reply:
Using such a wording now